pkgs/tools/graphics/yafaray-core/default.nix

```diff
@@ -31,7 +31,8 @@ stdenv.mkDerivation rec {
       homepage = "http://www.yafaray.org";
       maintainers = with maintainers; [ hodapp ];
       license = licenses.lgpl21;
-      platforms = [ "x86_64-linux" ];
+      platforms = [ "x86_64-linux" "aarch64-linux" ];
+      broken = !stdenv.isAarch64 && stdenv.isLinux; # at 2022-09-23
```

Comment:
Please add a comment with a short excerpt from the error like 'linking fails: undefined reference to ...'

Reply:
Done